projects/addon-editor/components/editor/editor.component.html

```diff
@@ -52,6 +52,8 @@
                 [editable]="interactive"
                 (valueChange)="onModelChange($event)"
                 (stateChange)="(0)"
+                (focusin)="control?.updateValueAndValidity()"
```

Comment:
Can we move this into `onActiveZone` so that it is not triggered when we navigate around toolbar/dropdowns?

Reply:
I check it

Reply:
fixed